packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -454,51 +487,84 @@ const SavedChartsHeader: FC = () => {
                                                 </Flex>
                                             </Menu.Target>
                                             <Menu.Dropdown>
-                                                {spaces?.map((spaceToMove) => {
-                                                    const isDisabled =
-                                                        savedChart?.spaceUuid ===
-                                                        spaceToMove.uuid;
-                                                    return (
-                                                        <Menu.Item
-                                                            key={
-                                                                spaceToMove.uuid
-                                                            }
-                                                            icon={
-                                                                <MantineIcon
+                                                {[
```

Comment:
Are the changes in this file from a different branch?

Reply:
Ups, yes, sorry. Let me fix it